cmd/manager/profileparser.go

```diff
@@ -8,6 +8,7 @@ import (
 	"os"
 
 	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
+	"github.com/antchfx/xmlquery"
 	"github.com/operator-framework/operator-sdk/pkg/log/zap"
 	"github.com/spf13/cobra"
 	"github.com/subchen/go-xmldom"
```

Comment:
go-xmldom is now only used in a struct definition that itself is unused. I wonder if we can get rid of it? There are some other uses that are still used for real, do you plan on getting rid of them as well?

Reply:
> go-xmldom is now only used in a struct definition that itself is unused. I wonder if we can get rid of it?

Good point, gotta delete that one!

> There are some other uses that are still used for real, do you plan on getting rid of them as well?
Not in this iteration, maybe in the future. The thing that I'm concerned about is that xmldom disregards XML namespaces, which is how we coded this whole thing; xmlquery cares about namespaces. In some old instances of the ARF, I noticed the namespace was randomized (they set something like `ns10` and stuff like this) so we might not be able to use xmlquery for ARF parsing unless that gets fixed in OpenSCAP.